src/Backend/Modules/Blog/DataFixtures/LoadBlogPosts.php

```diff
@@ -60,4 +65,26 @@ public function load(SpoonDatabase $database): void
             ]
         );
     }
+
+    private function insertCategory(): Category
+    {
+        $meta = new Meta(
+            'BlogCategory for tests', false,
+            'BlogCategory for tests', false,
+            'BlogCategory for tests', false,
+            'blogcategory-for-tests', false
+        );
+
+        $category = new Category(
+            Locale::fromString('en'),
+            'Blogcategory for functional tests',
+            $meta
+        );
+
+        $entityManager = $this->client->getContainer()->get('doctrine.orm.default_entity_manager');
```

Comment:
I thought we decided to doe $repo->add($entity)?

Reply:
WIP ;-)